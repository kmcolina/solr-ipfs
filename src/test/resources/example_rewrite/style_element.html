<html>
<head>
    <title>Style page</title>
    <link rel="stylesheet" type="text/css" href="styles/index_o1.css" />
</head>
<h1>Some styling</h1>

<div style="background: url(smileys_o20.png)">Hello</div>
<div style="background: url('one_o30.png'), url('two_o31.png')">World</div>
<p><a href="hello/" style="background-image: url(a_back_o40.png)"></a></p>
<p style="background: url(paragraph_o50.png)">paragraph</p>
</html>
