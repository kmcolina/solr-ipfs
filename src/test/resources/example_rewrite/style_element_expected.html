<html>
 <head>
  <title>Style page</title>
  <link rel="stylesheet" type="text/css" href="http://localhost:0000/solrwayback/services/view?source_file_path=somesourcefile&amp;offset=1">
 </head>
 <body>
  <h1>Some styling</h1>
  <div style="background: url(http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&amp;offset=20)">
   Hello
  </div>
  <div style="background: url('http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&amp;offset=30'), url('http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&amp;offset=31')">
   World
  </div>
  <p><a href="http://localhost:0000/solrwayback/services/web/20200430130700/http://example.com/somefolder/hello/" style="background-image: url(http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&amp;offset=40)"></a></p>
  <p style="background: url(http://localhost:0000/solrwayback/services/downloadRaw?source_file_path=somesourcefile&amp;offset=50)">paragraph</p>
 </body>
</html>